public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -108,6 +109,12 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
       notifyApp.error(conditionErrorMsg);
       return;
     }
+    // if this rule is using manual routing, we save this option in local storage
+    if (values.manualRouting) {
+      localStorage.setItem(MANUAL_ROUTING_KEY, 'true');
+    } else {
+      localStorage.removeItem(MANUAL_ROUTING_KEY);
```

Comment:
I might be overlooking something, shouldn't we explicitly set it to `false` here?

With the current implementation it looks like there's no way to make it default to `use notification policy` since the absence of the key means `use contact point`? Did I get that right?

Reply:
ops! you're right!

Reply:
updated!